gitbootcamp.rst

```diff
@@ -82,12 +82,19 @@ on the 2.7 release::
    $ git checkout -b 2.7 origin/2.7
 
 
-Deleting Local Branches
------------------------
+.. _deleting_branches:
+
+Deleting Branches
+-----------------
 
-To delete a branch that you no longer need::
+To delete a **local** branch that you no longer need::
 
-   $ git branch -D <branch-name>
```

Comment:
`git branch` supports both `-D` and `-d` (they are equivalent).
`git push` only supports `-d`.

Since it's easier to type and to remember, I switched it to the lowercase form.

Reply:
WWWWWHAT. They are not equivalent. git branch -d <branch_name> don't remove the branch and complains if the branch wasn't merged in another branch.

Reply:
Hmm they're not quite the same.
-D is the "force" option.
If a PR is not yet merged, -d will not delete the branch I think, you'll get an error.

Reply:
You are right, I misread the man page.  Apparently `-D` is equivalent to `-d --force` which deletes the branch even if it hasn't been merged.  Since usually branches get deleted after they have been merged, do you think it's ok to suggest `-d` anyway?  I can also add a note about deleting unmerged branches using `-D`.

Reply:
I've given up understanding when git will accept `-d`; it usually won't when I think it should.  I always use `-D`.

Reply:
Yep. I ended up using the uppercase `-D` because I don't want to deal with the error message :)

Reply:
I guess I'll revert the change to the uppercase `-D` then.